etc/migration/9.2-to-10.0/migration.cql

```diff
@@ -0,0 +1 @@
+ALTER "Principals" ADD "guest" boolean;
```

Comment:
Right, but that's a separate issue. The issue I'm trying to outline is when you want to share another item with the same email address. We will resolve the email address to the private user account. We need to know that user account is a guest or you won't be able to interact with it because of the visibility rules.